bin/build-drivers/build_drivers.clj

```diff
@@ -0,0 +1,22 @@
+(ns build-drivers
+  "Entrypoint for `bin/build-drivers.sh`. Builds all drivers, if needed."
+  (:require [build-drivers
+             [build-driver :as build-driver]
+             [common :as c]
+             [util :as u]]
+            [clojure.java.io :as io]
+            [colorize.core :as colorize]))
+
+(defn- build-drivers! []
```

Comment:
I'd be tempted to do `defstep` and have the docstring be also the step message and enforce one step per function (perhaps even doing something akin to continuation-passing for next step). This would also make it easier in the future to turn tasks into a DAG and eg. do parallel processing (like Tensorflow or Onyx does).

Reply:
The downside to having the message be the docstring would be that we couldn't create the message at runtime, e.g. 

```clj
(u/step (format "Delete file %s if exists" (pr-str file)) 
  (delete-file! ...))
```

So we lose a little bit of information if we do that. I suppose we *could* let the "docstring" be an arbitrary expression, but since Clojure docstrings come before arglists I think it would be confusing to have something like

```clj
(defstep delete-file! 
  (format "Delete file %s if exists" (pr-str file))
  [file]
  ...)
```

Reply:
I'm not 100% sure what the best way to have a `defstep` that would allow us to still create the messages at runtime would look like, so I'll think I'll leave things as-is for now until we come up with something. We can always change it later